recipes/norm/all/conanfile.py

```diff
@@ -67,8 +67,8 @@ def package_info(self):
         if self.settings.os == "Windows":
             self.cpp_info.system_libs = [
                 "ws2_32", "iphlpapi", "user32", "gdi32", "Advapi32", "ntdll"]
-        else:
-            self.cpp_info.system_libs = ["pthread"]
-
-        if self.settings.os == "Linux":
-            self.cpp_info.system_libs.extend(["dl", "rt"])
+        elif self.settings.os == "Linux":
+            self.cpp_info.system_libs = ["dl", "rt"]
+            
+        if self.settings.os == "Linux" or self.settings.os == "FreeBSD":
+            self.cpp_info.system_libs.append("pthread")
```

Comment:
If we have to change it lets get it right for both systems

```suggestion
        if self.settings.os in ["Linux", "FreeBSD"]:
            self.cpp_info.system_libs.extend(["dl", "rt", "pthread"])
```

Reply:
I think dl & rt are for Linux only.

Reply:
@ericLemanissier could you confirm?

Reply:
I have a doubt about rt, but dl is used on FreeBSD. Can you please add "FreeBSD" in the pull request description https://github.com/conan-io/conan-center-index/pull/4299#issue-557025012. That will make https://github.com/ericLemanissier/conan-center-index/actions?query=workflow%3ACI build your branch automatically on FreeBSD, so that we can see which system libs are actually needed. Thanks !

Reply:
When I found FreeBSD had all the API for the Realtime extensions I assumed it would be the same dynamic library

https://docs.oracle.com/cd/E86824_01/html/E54772/librt-3lib.html,
https://www.freebsd.org/cgi/man.cgi?query=sem&sektion=4&n=1

There's no package associate to the lib though!

***

Since FreeBSD is not a requirement can we do this?

```suggestion
        elif self.settings.os == "Linux":
            self.cpp_info.system_libs = ["pthread", "dl", "rt"]
```